vrpn_3DConnexion.h

```diff
@@ -79,6 +79,18 @@ class VRPN_API vrpn_3DConnexion: public vrpn_Button_Filter, public vrpn_Analog {
 };
 #endif  // not VRPN_USE_HID
 
+class VRPN_API vrpn_3DConnexionWireless : public vrpn_3DConnexion {
+public:
+	vrpn_3DConnexionWireless(vrpn_HidAcceptor *filter, unsigned num_buttons,
```

Comment:
is `vrpn_HidAcceptor` at least forward-declared even with `VRPN_USE_HID` not declared?